README.md

```diff
@@ -1 +1,46 @@
-# Restaurant_Page
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# restaurant-page
+
+> In this project, I built a simple website for a restaurant.
+
+## Built With
+
+- HTML
+- CSS3
+- JAVASCRIPT
+
+
+## Getting Started
```

Comment:
This project came after Webpack lesson, we need to use the knowledge in this project.

According to the pathwright, you need to add scripts that are necessary to build your project in package.json.

![image](https://user-images.githubusercontent.com/31897434/105122773-2333d500-5ad7-11eb-964a-567f5ceadc18.png)

https://microverse.pathwright.com/library/javascript-module-v10-copy/166786/path/step/97635163/